validation-test/Sema/type_checker_crashers_fixed/rdar98577451.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-typecheck-verify-swift -target %target-cpu-apple-macosx10.15
```

Comment:
You can just pass -disable-availability-checking instead and remove the REQUIRES: lines.

Reply:
On it!

Reply:
I wasn't sure whether it's supposed to work on Linux/Windows or not...

Reply:
https://github.com/apple/swift/pull/60580